File: SecondSem/CN/LAB/Python/ConnectionLessSimplex_2/UDPclient.py
import socket


def main():
    try:
        # Create a UDP socket
        client_socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)

        # Server address and port
        server_address = ("localhost", 9876)

        # Message to send
        message = "Hello, UDP Server!"

        # Convert the message to bytes
        send_data = message.encode()

        # Send the packet to the server
        client_socket.sendto(send_data, server_address)

        print(f"Message sent to server: {message}")

        # Close the socket
        client_socket.close()
    except Exception as e:
        print(e)


if __name__ == "__main__":
    main()

File: SecondSem/CN/LAB/Java/ConnectionLessSimplex_2/UDPClient.java
import java.net.DatagramPacket;
import java.net.DatagramSocket;
import java.net.InetAddress;

public class UDPClient {
    public static void main(String[] args) {
        try {
            // Create a DatagramSocket
            DatagramSocket clientSocket = new DatagramSocket();

            // Server address and port
            InetAddress serverAddress = InetAddress.getByName("localhost");
            int serverPort = 9876;

            // Message to send
            String message = "Hello, UDP Server!";

            // Convert the message to bytes
            byte[] sendData = message.getBytes();

            // Create a DatagramPacket to send data
            DatagramPacket sendPacket = new DatagramPacket(sendData, sendData.length, serverAddress, serverPort);

            // Send the packet to the server
            clientSocket.send(sendPacket);

            System.out.println("Message sent to server: " + message);

            // Close the socket
            clientSocket.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
